Ktisis/Interface/Components/Chara/CharaImportUI.cs

```diff
@@ -129,21 +139,23 @@ private void DrawMethodRadio(string label, LoadMethod method) {
 	// Mode selection
 	
 	public void DrawModesSelect() {
-		ImGui.Text("Appearance");
-		this.DrawModeSwitch("Body", SaveModes.AppearanceBody);
-		ImGui.SameLine();
-		this.DrawModeSwitch("Face", SaveModes.AppearanceFace);
-		ImGui.SameLine();
-		this.DrawModeSwitch("Hair", SaveModes.AppearanceHair);
-		
-		ImGui.Spacing();
-		
-		ImGui.Text("Equipment");
-		this.DrawModeSwitch("Gear", SaveModes.EquipmentGear);
-		ImGui.SameLine();
-		this.DrawModeSwitch("Accessories", SaveModes.EquipmentAccessories);
-		ImGui.SameLine();
-		this.DrawModeSwitch("Weapons", SaveModes.EquipmentWeapons);
+		using (ImRaii.Disabled(this.DisableModes)) {
```

Comment:
to reduce nesting you can use block level using, ex
```suggestion
		using var _ = ImRaii.Disabled(this.DisableModes);
```